homeassistant/components/freebox/router.py

```diff
@@ -89,7 +92,39 @@ async def update_all(self, now: datetime | None = None) -> None:
     async def update_device_trackers(self) -> None:
         """Update Freebox devices."""
         new_device = False
-        fbx_devices: list[dict[str, Any]] = await self._api.lan.get_hosts_list()
+
+        fbx_devices: list[dict[str, Any]] = []
+
+        # Access to Host list not available in bridge mode, API return error_code 'nodev'
+        if self.supports_hosts:
+            hosts_list_initialized = False
+            try:
+                fbx_devices.append(await self._api.lan.get_hosts_list())
```

Comment:
```suggestion
                fbx_devices = await self._api.lan.get_hosts_list()
```
You are adding a list of devices to a list, but we only want one list of devices

Tryed `.append` with ellipsis but don't seems to work.